Description: use locally installed mathjax
 This patch makes use of the locally installed mathjax script so that equations
 in the documentation can be rendered when no internet connection is available.
Author: Paul Brossier <piem@debian.org>
Forwarded: not-needed
Last-Update: 2013-12-29

--- a/doc/web.cfg
+++ b/doc/web.cfg
@@ -1526,7 +1526,7 @@
 # The default value is: http://cdn.mathjax.org/mathjax/latest.
 # This tag requires that the tag USE_MATHJAX is set to YES.
 
-MATHJAX_RELPATH        = https://cdn.mathjax.org/mathjax/latest
+MATHJAX_RELPATH        = file:///usr/share/javascript/mathjax
 
 # The MATHJAX_EXTENSIONS tag can be used to specify one or more MathJax
 # extension names that should be enabled during MathJax rendering. For example
